gulp/tasks/jscs.js

```diff
@@ -0,0 +1,21 @@
+(function() {
+    'use strict';
+
+    var jshint = require('gulp-jshint'),
```

Comment:
hey sorry for the drive-by @andy-armstrong , but as an interested linter, I looked at this PR. Should this file be called jshint.js? or, alternatively, should this statement be more like `var jscs = require('gulp-jscs')` ?